Wabbajack/Views/ModListGalleryView.xaml

```diff
@@ -139,4 +151,4 @@
             <iconPacks:PackIconMaterial Foreground="{Binding Foreground, RelativeSource={RelativeSource AncestorType={x:Type Button}}}" Kind="ArrowLeft" />
         </Button>
     </Grid>
-</rxui:ReactiveUserControl>
+</rxui:ReactiveUserControl>
```

Comment:
Git gets unhappy without an extra new line at the end.  Not sure why.
Smallest of nothing issues.. just happened to notice it was intentionally removed while scrolling through